instrumentation/database/sql/splunksql/driver_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package splunksql
+
+import (
+	"context"
+	"database/sql/driver"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
```

Comment:
IMO using `github.com/stretchr/testify/suite` is overkill here. I would prefer just to make some helper struct like

```go
type driverFixture struct {
	MockDriver *mockDriver
	OTelDriver *otelDriver
}

func newDriverFixture() {
	mockDrv := newMockDriver(nil)
	otelDrv := newDriver(mockDrv , newTraceConfig()).(*otelDriver)
	return  driverFixture{
		mockDriver: mockDrv,
		otelDriver: otelDrv,
	}
}
```

and add `fixture := newDriverFixture()` in the beginning of each test

Reply:
I'm going to keep the use of suite. IMO it is useful, remains consistent, and is working. We can refactor if we find an objective need in the future.